role/auditor/auditor.go

```diff
@@ -144,7 +147,9 @@ func (a *Auditor) Start(ctx context.Context) {
 
 				log.Debug().Bytes("task", task).Msg("performing validation")
 
-				result, err := mod.Validate(ctx, *input)
+				// TODO: auditor has universal client - config loaded from bootstrap.go
+				result, err := a.rpcClient.Call(ctx, *input)
```

Comment:
one thing I am still confused about, wont this call still be blocking becuase its waiting for the response from the RPC server?

is it just about making Validate occur on its own process or are we trying to run Validate async?

Reply:
Yes, it is blocking, but it's only blocking its own goroutine on line 142 so that's fine.
You would need to determine whether err is timeout error and log something different.